workspace/workflows/jobs.py

```diff
@@ -335,24 +349,49 @@ def _main(target: str, skip_successful: bool) -> str:
             )
         )
 
-    if target.count("/") > 1:
-        return report_invalid_target(target)
+    # Target might be a list or a single item
+    if "," in target:
```

Comment:
The `if` here isn't required (e.g. `"org".split(",") == ["org]`)

This could be managed in the parser args though.  
```
show_parser.add_argument('--target', nargs='+', default=[])
```
should parse `target` as a list. But, I think it expects the args to be non-comma separated, which may be complicated for our slack args.

You could also pass a [custom action ](https://docs.python.org/3/library/argparse.html#action) to the parser to do the splitting:
```
class SplitString(argparse.Action):
    def __call__(self, parser, namespace, values, option_string=None):
        setattr(namespace, self.dest, values.split(','))

show_parser.add_argument('--target', action=SplitString)
```
(This should probably do some stripping to handle being passed a quoted string with spaces)

Reply:
In fact, possibly a custom Action could do all the necessary work to return a list of (org, repo) pairs